src/game/Damage.cpp

```diff
@@ -1389,7 +1389,8 @@ void ARX_DAMAGES_DurabilityRestore(Entity * io, float percent) {
 			float mloss = 1.f;
 
 			if(io->ioflags & IO_ITEM) {
-				io->_itemdata->price -= util::to<long>(float(io->_itemdata->price) / io->max_durability);
+				io->_itemdata->buyPrice -= util::to<long>(float(io->_itemdata->buyPrice) / io->max_durability);
```

Comment:
why does this use util::to and then further down its using static_cast?

Reply:
no idea, probably they are aiming to switch every casting to `util::to` in the future

Reply:
guess you could use that then instead of the static_cast further down

Reply:
good catch, I've updated the code!